play-interop-test/src/main/scala/controllers/GreeterServiceImpl.scala

```diff
@@ -1,22 +1,22 @@
-package controllers
-
-import akka.NotUsed
-import akka.stream.Materializer
-import akka.stream.scaladsl.Source
-import example.myapp.helloworld.grpc.helloworld.{ GreeterService, HelloReply, HelloRequest }
-import javax.inject.{ Inject, Singleton }
-
-import scala.concurrent.Future
-
-/** Would be written by the user, with support for dependency injection etc */
-@Singleton
-class GreeterServiceImpl @Inject() (implicit mat: Materializer) extends GreeterService {
-
-  override def sayHello(in: HelloRequest): Future[HelloReply] = Future.successful(HelloReply(s"Hello, ${in.name}!"))
-
-  override def itKeepsTalking(in: Source[HelloRequest, NotUsed]): Future[HelloReply] = ???
-
-  override def itKeepsReplying(in: HelloRequest): Source[HelloReply, NotUsed] = ???
-
-  override def streamHellos(in: Source[HelloRequest, NotUsed]): Source[HelloReply, NotUsed] = ???
-}
+//package controllers
```

Comment:
I guess we have no choice but to split the play-interop-test into 2 subprojects, one for scala and one for java

Reply:
Eventually perhaps we might want to allow configuring code generation to generate some services as java and some as scala, but I'd say that's beyond MVC

Reply:
yes, i did 
split it up